charts/datarepo-api/templates/api-deployment.yaml

```diff
@@ -84,3 +85,15 @@ spec:
         {{ end -}}
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /var/secrets/google/sa-key.json
+# Supply the pod name: used to identify Stairway instances
+        - name: DATAREPO_PODNAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+# Share the timeout value between Kubernetes and Datarepo              
```

Comment:
line 93-98 could be defined in the value.yaml vs hard coding here. Example below


```
  env:
    GOOGLE_PROJECTID: broad-jade-dd
    DB_DATAREPO_USERNAME: drmanager
    SPRING_PROFILES_ACTIVE: google,cloudsql,dev,dd
    DB_STAIRWAY_USERNAME: drmanager
    DB_STAIRWAY_URI: jdbc:postgresql://dd-jade-gcloud-sqlproxy.dd:5432/stairway-dd
    DB_DATAREPO_URI: jdbc:postgresql://dd-jade-gcloud-sqlproxy.dd:5432/datarepo-dd
    DATAREPO_SHUTDOWNTIMEOUTSECONDS: 60
    DATAREPO_INKUBERNETES: true
```

Reply:
INKUBERNETES has to be set true, so I think it should stay.

It is important that Kubernetes deployed terminationGracePeriodSeconds is the same as DATAREPO_SHUTDOWNTIMEOUTSECONDS. That is why I made a value.yaml entry for it. So I'd have one place to set it and everything lines up.

I want to be able to set it on a per-deployment basis. I thought that if I overrode the  value `shutdownTimeoutSeconds` in ddDeployment.yaml, it would get applied in the template. Is that true?

Reply:
ddDeployment.yaml will override `shutdownTimeoutSeconds` so for clarity these environment vars will always be needed moving forward or should we add a 
```
{{ if .Values.shutdownTimeoutSeconds }}
```
then have the default value be empty so it doesn't render by default and when you need it you'd defined a time in seconds in your override values then its rendered?

Reply:
These are always needed moving forward.

Reply:
in that case LGTM